blivet/devices/partition.py

```diff
@@ -1033,12 +1033,14 @@ def populate_ksdata(self, data):
             if self.req_grow:
                 data.maxSizeMB = self.req_max_size.convert_to(MiB)
 
-            # data.disk = self.disk.name                      # by-id
             if self.req_disks and len(self.req_disks) == 1:
                 data.disk = self.disk.name
             data.primOnly = self.req_primary
         else:
-            data.onPart = self.name                     # by-id
+            if self.format and self.format.uuid:
+                data.onPart = "UUID=" + self.format.uuid
```

Comment:
@mmatsuya In your PR you are using `"disk/by-id/lvm-pv-uuid`, I think using the `UUID=<uuid>` here would be better -- it will cover other use cases than PVs too. If you don't have a specific use case for the `by-id` symlink I'd prefer this one.

Reply:
That looks great and better.
Just in cases, I tested it here and it worked. Thanks!